src/iota/components/HomeLayout/AppLibrariesSection.tsx

```diff
@@ -6,7 +6,7 @@ import {
   Streams,
   Stronghold,
   Wallet,
-} from '../../../icons';
+} from '../../../common/icons';
```

Comment:
Any reason why we not also use `@site` here?
```suggestion
} from '@site/src/common/icons';
```

Reply:
You are right, I missed a few, I'll check again.